Deploy/datares_test.py

```diff
@@ -0,0 +1,118 @@
+import base64
+import os
+import datetime
+from urllib.parse import quote as urlquote
+
```

Comment:
In the future I would rename this file to something more explanatory like app.py